apps/webpack.js

```diff
@@ -106,24 +109,39 @@ var karmaConfig = _.extend({}, baseConfig, {
 /**
  * Generate the appropriate webpack config based off of our base config and
  * some input options
+ * @param {string} uniqueName - Unique name for the bundle. Used by a webpack
+ *   option to make sure our different bundles don't end up sharing webpack runtimes
  * @param {object} options
  * @param {string} options.output
  * @param {string[]} options.entries - list of input source files
+ * @param {string} commonFile
  * @param {bool} options.minify
  * @param {bool} options.watch
  * @param {string} options.piskelDevMode
+ * @param {string[]} options.provides - list of "external" modules that this
+ *   bundle actually provides (and thus should not be external here)
  */
 function create(options) {
+  var uniqueName = options.uniqueName;
   var outputDir = options.output;
   var entries = options.entries;
+  var commonFile = options.commonFile;
   var minify = options.minify;
   var watch = options.watch;
   var piskelDevMode = options.piskelDevMode;
+  var provides = options.provides;
+
+  if (!uniqueName) {
```

Comment:
Pretty easily, though perhaps at the expense of being meaningful (i.e. jsonp_1). Might be we don't care about it being meaningful though.